6/Dockerfile

```diff
@@ -29,14 +37,20 @@ LABEL summary="$SUMMARY" \
       io.k8s.display-name="Node.js $NODEJS_VERSION" \
       io.openshift.expose-services="8080:http" \
       io.openshift.tags="builder,nodejs,nodejs$NODEJS_VERSION" \
+      io.openshift.s2i.scripts-url="image:///usr/local/s2i" \
       com.redhat.dev-mode="DEV_MODE:false" \
       com.redhat.deployments-dir="/opt/app-root/src" \
       com.redhat.dev-mode.port="DEBUG_PORT:5858"\
       com.redhat.component="rh-nodejs6-docker" \
       name="centos/nodejs-6-centos7" \
       version="6" \
       release="1" \
-      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      help="For more information visit https://github.com/container-images/nodejs" \
+      usage="s2i build <SOURCE-REPOSITORY> nodejs:6 <APP-NAME>
+
+# Defines the location of S2I
+COPY ./.s2i/bin/ /usr/local/s2i
```

Comment:
A similar command is already present a couple more lines later in the Dockerfile. The difference being that the scripts are installed to `/usr/libexec/s2i` (as defined in https://github.com/sclorg/s2i-base-container/blob/master/core/Dockerfile)